lib/java/sources-parser/src/main/java/co/airy/sources-parser/SourcesParser.java

```diff
@@ -0,0 +1,60 @@
+package co.airy.sources_parser;
```

Comment:
This is something I missed on `ibm-watson-assistant` and it was causing weird behaviour. the source path should always be snake case.
```
lib/java/sources-parser/src/main/java/co/airy/sources_parser/SourcesParser.java
```

`sources_parser` not `sources-parser`

See https://github.com/airyhq/airy/commit/1af02ba86b233b9bebaa08133d3cd8502f71af3f